net/service/handlers.go

```diff
@@ -1132,7 +1132,7 @@ func (service *OpenBazaarService) handleOrderCompletion(p peer.ID, pmes *pb.Mess
 		outValue := big.NewInt(0)
 		for _, r := range records {
 			if !r.Spent && r.Value.Cmp(big.NewInt(0)) > 0 {
-				outpointHash, err := hex.DecodeString(ut.NormalizeAddress(r.Txid))
+				outpointHash, err := hex.DecodeString(r.Txid)
```

Comment:
Does this ever have 0x in the txid? If so I think this will return an error no?

Reply:
If there is a 0x prefix, there will be an error. Since the Txid itself is a chainhash.Hash (as returned by the wallet), there is no such possibility.